hubblestack_nova_profiles/cis/amazon-level-1-scored-v1-0-0.yaml

```diff
@@ -0,0 +1,814 @@
+      Amazon Linux*:
+      - net.ipv4.conf.all.accept_source_route:
+          match_output: '0'
+          tag: CIS-4.2.1
+      - net.ipv4.conf.default.accept_source_route:
+          match_output: '0'
+          tag: CIS-4.2.1
+    description: Disable Source Routed Packet Acceptance (Scored)
+  tcp_syn_cookies:
+    data:
+      Amazon Linux*:
+      - net.ipv4.tcp_syncookies:
+          match_output: '1'
+          tag: CIS-4.2.8
+    description: Enable TCP SYN cookies (Scored)
+
+
+grep:
+  whitelist:
+
+    fstab_tmp_partition:
+      data:
+        Amazon Linux*:
+          - '/etc/fstab':
+              tag: 'CIS-1.1.1'
+              pattern: '/tmp'
+      description: 'Create Separate Partition for /tmp (Scored)'
+
+    fstab_tmp_partition_nodev:
+      data:
+        Amazon Linux*:
+          - '/etc/fstab':
+              tag: 'CIS-1.1.2'
+              pattern: '/tmp'
+              match_output: 'nodev'
+      description: 'Set nodev option for /tmp Partition (Scored)'
+
+    fstab_tmp_partition_nosuid:
+      data:
+        Amazon Linux*:
+          - '/etc/fstab':
+              tag: 'CIS-1.1.3'
+              pattern: '/tmp'
+              match_output: 'nosuid'
+      description: 'Set nosuid option for /tmp Partition (Scored)'
+
+    fstab_tmp_partition_noexec:
+      data:
+        Amazon Linux*:
+          - '/etc/fstab':
+              tag: 'CIS-1.1.4'
+              pattern: '/tmp'
+              match_output: 'noexec'
+      description: 'Set noexec option for /tmp Partition (Scored)'
+
+    fstab_var_partition:
+      data:
+        Amazon Linux*:
+          - '/etc/fstab':
+              tag: 'CIS-1.1.5'
+              pattern: '/var'
+      description: 'Create Separate Partition for /var (Scored)'
+
+    fstab_var_tmp_bind_mount:
+      data:
+        Amazon Linux*:
+          - '/etc/fstab':
+              tag: 'CIS-1.1.6'
+              pattern: '/tmp'
+              match_output: '/var/tmp'
+      description: 'Bind Mount the /var/tmp directory to /tmp (Scored)'
+
+    fstab_var_log_partition:
+      data:
+        Amazon Linux*:
+          - '/etc/fstab':
+              tag: 'CIS-1.1.7'
+              pattern: '/var/log'
+      description: 'Create Separate Partition for /var/log (Scored)'
+
+    fstab_var_log_audit_partition:
+      data:
+        Amazon Linux*:
+          - '/etc/fstab':
+              tag: 'CIS-1.1.8'
+              pattern: '/var/log/audit'
+      description: 'Create Separate Partition for /var/log/audit (Scored)'
+
+    fstab_home_partition:
+      data:
+        Amazon Linux*:
+          - '/etc/fstab':
+              tag: 'CIS-1.1.9'
+              pattern: '/home'
+      description: 'Create Separate Partition for /home (Scored)'
+
+    fstab_home_partition_nodev:
+      data:
+        Amazon Linux*:
+          - '/etc/fstab':
+              tag: 'CIS-1.1.10'
+              pattern: '/home'
+              match_output: 'nodev'
+      description: 'Add nodev Option to /home (Scored)'
```

Comment:
As per CIS doc, 'If a /home partition exists run the following command and verify that the nodev option is set on /home.' but we are not checking for /home partition in via 'mount | grep /home'. So I think we should check 'mount | grep /home' first and if /home present then we should check /home pattern in /etc/fstab file.